<launch>
  <arg name="image_topic" default="/camera/image_raw"/>
  <arg name="filtered_image_topic" default="/filtered_image"/>
 
  <node pkg="blob_segmentation" name="segment_blob" type="segment_blob.py" output="screen">
    <param name="image_topic" type="string" value="$(arg image_topic)" />
    <param name="filtered_image_topic" type="string" value="$(arg filtered_image_topic)" />
  </node>

</launch>
